docs/content/stable/secure/authorization/rbac-model.md

```diff
@@ -44,6 +44,64 @@ Roles in YSQL can represent individual users or a group of users. They encapsula
 
 * Roles inherit the privileges of all other roles granted to them.
 
```

Comment:
Be sure to incorporate comments from the corresponding files above in /latest :)